manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162

```

```suggestion
PackageVersion: 2.0
```
Writes `2.0` to the control panel when the install works